meta-mender-core/classes/mender-helpers.bbclass

```diff
@@ -259,7 +259,18 @@ def mender_get_extra_parts_offset(d):
     # we have no boot part and add two or more extra partitions then move index as extended partition is created
     if data_num < 4 and total_part_num >= 5: extra_part_num += 1
 
-    return extra_part_num 
+    return extra_part_num
+
+def mender_get_extra_parts_offset_by_id(d, id = None):
```

Comment:
I can't see that this is used anywhere.

Reply:
I needed this functionality in my own integration. 

Since the EXTRA_PARTS doesn't have hard-coded partition numbers, I needed someway to figure out what they are.